system-tests/projects/react17/package.json

```diff
@@ -7,7 +7,7 @@
     "@vitejs/plugin-react": "^1.3.2",
     "css-loader": "^4.3.0",
     "style-loader": "^2.0.0",
-    "vite": "2.9.14",
+    "vite": "3.2.2",
```

Comment:
Why was this updated? Should bump `@vitejs/plugin-react` as well

Reply:
Some context here in slack: https://cypressio.slack.com/archives/C011BAPC614/p1667810101355419?thread_ts=1667789779.018379&cid=C011BAPC614